themes/app/templates/Includes/Header.ss

```diff
@@ -1,5 +1,6 @@
 <header>
     <main-nav>
+        <img slot="logo" class="main-nav__logo" src="$ThemeDir/dist/images/logo.svg" alt="Logo" />
```

Comment:
- Replace with user defined image from CMS
- Use the picture element to restrict height of image displayed 
🙂

Reply:
Will do 👌🏻